### PR TITLE
OneDrive Sync Agent Health Report Changes

### PR DESCRIPTION
**Pull request by maurice-daly**
Updated the documentation to advise the old sync agent health setting has been retired, and included instructions as to how to set this in Intune.

### CODE REVIEW

File: SharePoint/SharePointOnline/sync-health.md [hunk 1]
```diff
@@ -74,51 +76,128 @@ This tab provides how-to steps for enabling sync reports on Windows devices.
     > [!NOTE]
     > When you generate a new key for the first time, it can take up to 30 seconds for it to appear.
 
-6. Enable the OneDrive EnableSyncAdminReports Group Policy Object (GPO).
+6. Enable the OneDrive EnableSyncAdminReports setting via Group Policy Object or Intune.
+
+> [!IMPORTANT]
+> **You must enable this setting on the devices from which you want to get reports using one of the below methods.** This setting does not affect users. We recommend a gradual rollout starting with a few test devices per day, then up to 100 devices per day, then gradually up to 10,000 devices per day until you finish.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sync-health.md:82]:
```suggestion
> **You must enable this setting on the devices from which you want to get reports using one of the following methods.** This setting does not affect users. We recommend a gradual rollout starting with a few test devices per day, then up to 100 devices per day, then gradually up to 10,000 devices per day until you are done.
```

File: SharePoint/SharePointOnline/sync-health.md [hunk 2]
```diff
@@ -74,51 +76,128 @@ This tab provides how-to steps for enabling sync reports on Windows devices.
...
+
+### Group Policy
+
+You can enable this setting in multiple ways:
+
+- Edit the registry
+
+    a. Go to HKLM\SOFTWARE\Policies\Microsoft\OneDrive
+
+    b. Right-click > **New** > **DWORD (32-bit) Value**
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sync-health.md:92]:
```suggestion
    b. **Right-click** > **New** > **DWORD (32-bit) Value**
```

File: SharePoint/SharePointOnline/sync-health.md [hunk 3]
```diff
@@ -74,51 +76,128 @@ This tab provides how-to steps for enabling sync reports on Windows devices.
...
+
+- Open Group Policy Editor (gpedit.msc).
+
+- Go to Computer Configuration\Administrative Templates\OneDrive.
 
-    - Edit the registry
+- Choose **Enable sync health reporting for OneDrive**.
 
-        a. Go to HKLM\SOFTWARE\Policies\Microsoft\OneDrive
+- Select **Enabled** and then press **OK**.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sync-health.md:118]:
```suggestion
- Select **Enabled** and then select **OK**.
```

File: SharePoint/SharePointOnline/sync-health.md [hunk 4]
```diff
@@ -74,51 +76,128 @@ This tab provides how-to steps for enabling sync reports on Windows devices.
...
+  
+### Intune
 
-        b. Right-click > **New** > **DWORD (32-bit) Value**
+To apply the setting through Intune, we recommend that you use the "Settings Catalog" profile type:
 
-        c. Name: EnableSyncAdminReports
+-  Go to [Microsoft Intune Admin Center](https://intune.microsoft.com) and sign in as a global admin, Intune admin role, or an account with sufficient RBAC permissions.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sync-health.md:124]:
```suggestion
-  Go to [Microsoft Intune Admin Center](https://intune.microsoft.com) and sign in as a global admin, Intune admin, or an account with sufficient RBAC permissions.
```

File: SharePoint/SharePointOnline/sync-health.md [hunk 5]
```diff
@@ -74,51 +76,128 @@ This tab provides how-to steps for enabling sync reports on Windows devices.
...
 
-        d. Type: REG_DWORD
+-  Click on Devices, Windows, and then click on "Configuration profiles".
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sync-health.md:126]:
```suggestion
-  Select **Devices** > **Windows**, and then select **Configuration profiles**.
```

File: SharePoint/SharePointOnline/sync-health.md [hunk 6]
```diff
@@ -74,51 +76,128 @@ This tab provides how-to steps for enabling sync reports on Windows devices.
...
 
-        e. Data: 1
+-  Edit or create a new OneDrive profile (Platform - "Windows 10 and later", Profile type - "Settings Catalog").
 
-        :::image type="content" source="media/createregkey.png" alt-text="Screenshot image depicting the creation of a new key in the Registry Editor":::
+-  If creating a new profile, provide a name for the profile and go to the Configuration settings tab. If editing a profile, you will by default go direct to the Configuration settings tab.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sync-health.md:130]:
```suggestion
-  If creating a new profile, provide a name for the profile and go to the Configuration settings tab. If editing a profile, you will by default go directly to the Configuration settings tab.
```

File: SharePoint/SharePointOnline/sync-health.md [hunk 7]
```diff
@@ -74,51 +76,128 @@ This tab provides how-to steps for enabling sync reports on Windows devices.
...
 
-    - Run Command Prompt as an administrator, and then run the following command:
+-  Click on "Add setting" and the Settings picker will be displayed on the right side of the UI.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sync-health.md:132]:
```suggestion
-  Select **Add setting** and the Settings picker will be displayed on the right side of the UI.
```

File: SharePoint/SharePointOnline/sync-health.md [hunk 8]
```diff
@@ -74,51 +76,128 @@ This tab provides how-to steps for enabling sync reports on Windows devices.
...
-       ```PowerShell
-        reg.exe add HKLM\Software\Policies\Microsoft\OneDrive /v EnableSyncAdminReports /t REG_DWORD /d 1
-       ```
+-  In the search bar, add "OneDrive". Alternatively you can scroll down to the OneDrive settings.
 
-    - Use [Group Policy](use-group-policy.md#manage-onedrive-using-group-policy).
+-  Click on the checkbox next to the "Enable sync health reporting for OneDrive" setting name.
 
-    To apply the setting on a single PC, follow these steps:
+-  Close the Settings picker from the "X" at the top right.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sync-health.md:138]:
```suggestion
-  Close the Settings picker from the **X** at the top right.
```

File: SharePoint/SharePointOnline/sync-health.md [hunk 9]
```diff
@@ -74,51 +76,128 @@ This tab provides how-to steps for enabling sync reports on Windows devices.
...
+=======
 
-    - Go to Computer Configuration\Administrative Templates\OneDrive.
+- Edit the registry
 
-    - Choose **Enable sync health reporting for OneDrive**.
+    a. Go to HKLM\SOFTWARE\Policies\Microsoft\OneDrive
 
-    - Select **Enabled** and then press **OK**.
+    b. Right-click > **New** > **DWORD (32-bit) Value**
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sync-health.md:146]:
```suggestion
    b. **Right-click** > **New** > **DWORD (32-bit) Value**
```

File: SharePoint/SharePointOnline/sync-health.md [hunk 10]
```diff
@@ -74,51 +76,128 @@ This tab provides how-to steps for enabling sync reports on Windows devices.
...
+
+- Run Command Prompt as an administrator, and then run the following command:
+
+   ```PowerShell
+    reg.exe add HKLM\Software\Policies\Microsoft\OneDrive /v EnableSyncAdminReports /t REG_DWORD /d 1
+   ```
+
+- Use [Group Policy](use-group-policy.md#manage-onedrive-using-group-policy).
+
+To apply the setting on a single PC, follow these steps:
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sync-health.md:164]:
```suggestion
To apply the setting on a single device, follow these steps:
```

File: SharePoint/SharePointOnline/sync-health.md [hunk 11]
```diff
@@ -74,51 +76,128 @@ This tab provides how-to steps for enabling sync reports on Windows devices.
...
+
+- Open Group Policy Editor (gpedit.msc).
+
+- Go to Computer Configuration\Administrative Templates\OneDrive.
+
+- Choose **Enable sync health reporting for OneDrive**.
+
+- Select **Enabled** and then press **OK**.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sync-health.md:172]:
```suggestion
- Select **Enabled** and then select **OK**.
```

File: SharePoint/SharePointOnline/sync-health.md [hunk 12]
```diff
@@ -74,51 +76,128 @@ This tab provides how-to steps for enabling sync reports on Windows devices.
...
+  
+### Intune
+
+To apply the setting through Intune, we recommend that you use the "Settings Catalog" profile type:
+
+-  Go to [Microsoft Intune Admin Center](https://intune.microsoft.com) and sign in as a global admin, Intune admin role, or an account with sufficient RBAC permissions.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sync-health.md:178]:
```suggestion
-  Go to [Microsoft Intune Admin Center](https://intune.microsoft.com) and sign in as a global admin, Intune admin, or an account with sufficient RBAC permissions.
```

File: SharePoint/SharePointOnline/sync-health.md [hunk 13]
```diff
@@ -74,51 +76,128 @@ This tab provides how-to steps for enabling sync reports on Windows devices.
...
+
+-  Click on Devices, Windows, and then click on "Configuration profiles".
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sync-health.md:180]:
```suggestion
-  Select **Devices** > **Windows**, and then select **Configuration profiles**.
```

File: SharePoint/SharePointOnline/sync-health.md [hunk 14]
```diff
@@ -74,51 +76,128 @@ This tab provides how-to steps for enabling sync reports on Windows devices.
...
+
+-  Edit or create a new OneDrive profile (Platform - "Windows 10 and later", Profile type - "Settings Catalog").
+
+-  If creating a new profile, provide a name for the profile and go to the Configuration settings tab. If editing a profile, you will by default go direct to the Configuration settings tab.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sync-health.md:184]:
```suggestion
-  If creating a new profile, provide a name for the profile and go to the Configuration settings tab. If editing a profile, you will by default go directly to the Configuration settings tab.
```

File: SharePoint/SharePointOnline/sync-health.md [hunk 15]
```diff
@@ -74,51 +76,128 @@ This tab provides how-to steps for enabling sync reports on Windows devices.
...
+
+-  Click on "Add setting" and the Settings picker will be displayed on the right side of the UI.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sync-health.md:186]:
```suggestion
-  Select **Add setting** and the Settings picker will be displayed on the right side of the UI.
```

File: SharePoint/SharePointOnline/sync-health.md [hunk 16]
```diff
@@ -74,51 +76,128 @@ This tab provides how-to steps for enabling sync reports on Windows devices.
...
+
+-  In the search bar, add "OneDrive". Alternatively you can scroll down to the OneDrive settings.
+
+-  Click on the checkbox next to the "Enable sync health reporting for OneDrive" setting name.
+
+-  Close the Settings picker from the "X" at the top right.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sync-health.md:192]:
```suggestion
-  Close the Settings picker from the **X** at the top right.
```

File: SharePoint/SharePointOnline/sync-health.md [hunk 17]
```diff
@@ -74,51 +76,128 @@ This tab provides how-to steps for enabling sync reports on Windows devices.
...
+
+-  Click on the toggle switch to set "Enable sync health reporting for OneDrive" to "Enabled".
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sync-health.md:194]:
```suggestion
-  Select the toggle switch to set "Enable sync health reporting for OneDrive" to **Enabled**.
```

File: SharePoint/SharePointOnline/sync-health.md [hunk 18]
```diff
@@ -74,51 +76,128 @@ This tab provides how-to steps for enabling sync reports on Windows devices.
...
+
+-  If creating a new profile, continue through the wizard, assing scope tags if required, and assigning the profile as required. If you are editing the profile, click on the "Review + save" button.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sync-health.md:196]:
```suggestion
-  If creating a new profile, continue through the step-by-step guide, assigning scope tags if required, and assigning the profile as required. If you are editing the profile, select **Review + save**.
```

File: SharePoint/SharePointOnline/sync-health.md [hunk 19]
```diff
@@ -74,51 +76,128 @@ This tab provides how-to steps for enabling sync reports on Windows devices.
...
+-  In the search bar, add "OneDrive". Alternatively you can scroll down to the OneDrive settings.
 
-    - Use [Group Policy](use-group-policy.md#manage-onedrive-using-group-policy).
+-  Click on the checkbox next to the "Enable sync health reporting for OneDrive" setting name.
 
-    To apply the setting on a single PC, follow these steps:
+-  Close the Settings picker from the "X" at the top right.
 
-    - Open Group Policy Editor (gpedit.msc).
+=======
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sync-health.md:140]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @maurice-daly, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by maurice-daly** [comment]
Just checking in for an approval on the pull request.

**Comment by JohanFreelancer9** [comment]
> Just checking in for an approval on the pull request.

@scanum

**Comment by maurice-daly** [comment]
Just another push on the commit.

**Comment by scanum** [comment]
@MikePlumleyMSFT 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by MikePlumleyMSFT** [comment]
@maurice-daly there's another update in the queue to clarify that the he old sync agent health setting has been retired. Intune is documented separately. We wouldn't include that here.

**Pull request closed without merging**